build.sh

```diff
@@ -451,6 +520,8 @@ __SkipRestore=""
 __SkipNuget=0
 __SkipCoreCLR=0
 __SkipMSCorLib=0
+# The default value of __SkipTests is off temporarily.
+__SkipTests=1
```

Comment:
@mkborg Currently there is no option to enable the test building in build.sh scripts. Instead you can enable it manully by updating __SkipTests=0. Until now, there are some build error issue in test building so I couldn't enable the option yet. After fixing whole issues, I will enable. Apart from this, I thought about adding new option like testsOnly to enable only tests build but as I know tests binaries are very relevant to coreclr libraries. So it didn't add it.